src/upcoming-components/Feature/Feature.stories.tsx

```diff
@@ -5,6 +5,7 @@ import React from 'react';
 import { Feature, Props } from './Feature';
 import Heading from '../../components/Heading';
 import Text from '../../components/Text';
+import PlaceholderImage from '../../static/compass-solo.svg';
```

Comment:
> So one question is – should the static folder:
> - be at the top level, above src and .storybook
> - in src and the recipes reach into this folder (just like they pull components out of src)
> - be duplicated like this in both places

I think it's fine to leave it in .storybook/static, with the other stuff in there. Or can we not import from there for some reason?

Reply:
Looks like there's a lint check against that:
```
/Users/drater/edu-design-system/src/upcoming-components/Feature/Feature.stories.tsx
  6:1  error  '../../../.storybook/static/compass-solo.svg' import is restricted from being used by a pattern. Do not import files from `.storybook/` into `src`. If needed, move files into `src/storyUtils` instead  no-restricted-imports
```
🤔

Reply:
PR: https://github.com/chanzuckerberg/edu-design-system/pull/716
Shortcut: https://app.shortcut.com/czi-edu/story/172777/add-lint-error-against-importing-from-storybook-or-docs-into-src

Reply:
I think generally the assets should be left in `.storybook/` for use in recipes and pages and components can have fpo blocks (ideally), but for this case would make sense to have actual assets. 

I vote for importing from `.storybook/` and using a eslint ignore for this case

Reply:
Interesting. Looks like that was introduced in #716, but the Slack thread with context is lost to the sands of time.

@anniehu4, do you remember why we added the no-restricted-imports item for `.storybook/`?

Reply:
sorry I'm so late to this 😓  @dierat @jinlee93 @ahuth I did some hunting and I believe it was related to https://github.com/chanzuckerberg/edu-design-system/pull/713. Essentially:
- the way the build used to be set up, `tsc` would compile everything in `src/` (including stories)
- then we'd remove the stories in a post-processing step
- because of that, importing anything from `.storybook` into `src` would inadvertently also compile `.storybook`, which broke the output structure

However, now we have a separate TS config for building the project vs. local development. We only build components listed in `src/index.ts`. I think it would be fine to:
- get rid of the lint rule now
- keep these images in `.storybook/static`
- verify that `yarn build` outputs the same structure into `lib` (since that was the original problem)

Reply:
Thanks @anniehu4 ! I imported the image into a component in the `components` directory and didn't see any change in structure in the `lib` directory after running `yarn build`, so I guess we're in the clear?

Reply:
sgtm!